resources/hvac.rb

```diff
@@ -369,6 +369,9 @@ def self.apply_evaporative_cooler(model, runner, frac_cool_load_served,
     evap_cooler = OpenStudio::Model::EvaporativeCoolerDirectResearchSpecial.new(model, model.alwaysOnDiscreteSchedule)
     evap_cooler.setName(obj_name)
     evap_cooler.setCoolerEffectiveness(cooler_effectiveness)
+    evap_cooler.setEvaporativeOperationMinimumDrybulbTemperature(0) # relax limitation to open evap cooler for any potential cooling
+    evap_cooler.setEvaporativeOperationMaximumLimitWetbulbTemperature(50) # relax limitation to open evap cooler for any potential cooling
+    evap_cooler.setEvaporativeOperationMaximumLimitDrybulbTemperature(50) # relax limitation to open evap cooler for any potential cooling
```

Comment:
These three limitations caused evap cooler closed for some available hours. This turned out to be the main reason causing aggregated fan energy inconsistent with metered fan energy.